querybook/server/datasources/metastore.py

```diff
@@ -671,25 +669,18 @@ def get_schemas(metastore_id, limit=5, offset=0, sort_key="name", sort_order="de
     "/table/<schema_name>/<table_name>/sync/",
     methods=["PUT"],
 )
-def sync_table_from_metastore(
-    schema_name, table_name, metastore_id, is_delete: bool = False
-):
-    """Sync table info from metastore. Delete the table if is_delete is True.
+def sync_table_from_metastore(schema_name, table_name, metastore_id):
```

Comment:
nit: since the code between refresh and sync are so similar, can you update the name to be more consistent?

like
`sync_table_by_table_id`
`sync_table_by_name`